index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Night of Recovery Concert!!!</title>
+  </head>
+  <body id="body">
+    <header>
+      <section class="container-fluid top-header">
+        <div class="container">
+          <div
+            class="topHeader d-none d-md-flex justify-content-end align-items-center p-0 m-0"
+          >
+            <i class="fa fa-facebook" aria-hidden="true"></i>
+            <i class="fa fa-twitter" aria-hidden="true"></i>
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Login</a>
+          </div>
+        </div>
+      </section>
+      <menu class="header-menu">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+          <div class="container">
+            <a class="navbar-brand d-none d-md-block" href="index.html"
+              ><img src="images/logo.png" alt="concert logo"
+            /></a>
+            <button
+              class="navbar-toggler"
+              type="button"
+              data-bs-toggle="collapse"
+              data-bs-target="#navbarSupportedContent"
+              aria-controls="navbarSupportedContent"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div
+              class="collapse navbar-collapse justify-content-end"
+              id="navbarSupportedContent"
+            >
+              <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                <div class="close-btn">
+                  <i class="fa fa-close closeBtn" aria-hidden="true"></i>
+                </div>
+                <li class="nav-item">
+                  <a
+                    class="nav-link active"
+                    aria-current="page"
+                    href="about.html"
+                    >About</a
```

Comment:
- Hey, great job making the changes as requested by the previous reviewer, however, ther's a lot issues you still need to address to improve your HTML semantics and layout. kindly make all the changes I'd point out to improve your semantic usage below:

   - You've wrongly used the `<menu>` tag here, kindly see this [link about menu](https://www.w3schools.com/tags/tag_menu.asp) to know how to use them correctly as you're not supposed to use it here. kindly remove the `<menu>` tag, instead, since you basically used it as a container holding the bottom nav, consider using a `<div>` tag.
   - Also, as pointed out by the previous reviewer, you are not supposed to use the section tag here in the header as you've used. Please, kindly remove it or update it to a div since you've used it for the purpose of a container. please, fix this issue.